lib/tether/tether.go

```diff
@@ -649,6 +607,42 @@ func (t *tether) handleSessionExit(session *SessionConfig) {
 	}
 }
 
+func (t *tether) loggingLocked(session *SessionConfig) error {
+	var writer syslog.Writer
+
+	stdout, stderr, err := t.ops.SessionLog(session)
+	if err != nil {
+		detail := fmt.Errorf("failed to get log writer for session: %s", err)
+		session.Started = detail.Error()
+
+		return detail
+	}
+	session.Outwriter = stdout
+	session.Errwriter = stderr
+	session.Reader = dio.MultiReader()
+
+	if session.Diagnostics.SysLogConfig != nil {
+		cfg := session.Diagnostics.SysLogConfig
+		var w syslog.Writer
+		if writer == nil {
+			writer, err = syslog.Dial(cfg.Network, cfg.RAddr, syslog.Info|syslog.Daemon, fmt.Sprintf("%s", t.config.ID[:shortLen]))
```

Comment:
Grrr I was planning to move  var writer syslog.Writer into tether struct to keep that behavior but looks like I forgot that, changing now